packages/docs/docs/website/static/css/custom.css

```diff
@@ -1,6 +1,32 @@
 /* your custom css */
 
+@media (max-width: 480px){ 
+  .docMainWrapper .wrapper{ 
+    padding:0 20px;
+ 
+  }
+
+  .navigationSlider .slidingNav ul{ 
+    background-color: #1d2327 !important;
+      }
+.navigationSlider .slidingNav ul li a {
+    padding: 18px 10px 0 10px !important;
+}
+
+  }
+
+
+
 @media (max-width: 735px) {
+    .docMainWrapper .wrapper{ 
+    padding:0 20px;
+ 
+  }
+
+  .navigationSlider .slidingNav ul{ 
+    background-color: #1d2327 !important;
+      }
```

Comment:
I think it is not properly indented, at least my browser is showing me these lines weirdly indented. Would you mind taking a look at it?